anki/stdmodels.py

```diff
@@ -10,7 +10,7 @@
 # Basic
 ##########################################################################
 
-def addBasicModel(col):
+def newBasicModel(col):
```

Comment:
Should this be prefixed with an underscore?

Reply:
Good idea. Makes sens. Done in a second commit